src/editor/data/previews/header-block.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
```

Comment:
it is because [here](https://github.com/gravity-ui/page-constructor/pull/347/files#diff-13be0aa655e665e6dbb5d47046fad964040d9aaa12e3bd0c58ace31dff971152R22) we require block preview svg based on block's name (I think we need to refactor componentsMap the same not to add blocks to it manually)

Reply:
> Why is this file named as header-block?

it is because [here](https://github.com/gravity-ui/page-constructor/pull/347/files#diff-13be0aa655e665e6dbb5d47046fad964040d9aaa12e3bd0c58ace31dff971152R22) we require block preview svg based on block's name (I think we need to refactor componentsMap the same not to add blocks to it manually)

Reply:
Maybe we can add some prefix? It would help to understand what a file is that and not to confuse about real block-files.